webpack/JobWizard/steps/AdvancedFields/AdvancedFields.js

```diff
@@ -98,6 +99,14 @@ export const AdvancedFields = ({ advancedValues, setAdvancedValues }) => {
             })
           }
         />
+        <ExecutionOrderingField
+          isRandomizedOrdering={advancedValues.isRandomizedOrdering}
+          setValue={newValue =>
+            setAdvancedValues({
+              isRandomizedOrdering: newValue,
+            })
```

Comment:
Suggestion:
```suggestion
          setValue={isRandomizedOrdering => setAdvancedValues({ isRandomizedOrdering })
```

Reply:
I'm thinking to destuct `advancedValues` in a different pr maybe, so there would be a const named `isRandomizedOrdering`